cli/download_source.go

```diff
@@ -57,7 +56,7 @@ func downloadTerraformSource(source string, terragruntOptions *options.Terragrun
 	}
 
 	terragruntOptions.Logger.Printf("Copying files from %s into %s", terragruntOptions.WorkingDir, terraformSource.WorkingDir)
-	if err := util.CopyFolderContents(terragruntOptions.WorkingDir, terraformSource.WorkingDir); err != nil {
+	if err := util.CopyFolderContents(terragruntOptions.WorkingDir, terraformSource.WorkingDir, ".terragrunt-module-manifest"); err != nil {
```

Comment:
`".terragrunt-module-manifest"` should be stored in a `const`

Reply:
Yes, I was thinking about this too. Sounds reasonable, will fix that